typelib/value_ops.cc

```diff
@@ -651,7 +651,8 @@ struct FDOutputStream : public OutputStream
 
     void write(boost::uint8_t const* data, size_t size)
     {
-        ::write(fd, data, size);
+        ssize_t ignored; (void) ignored;
+        ignored = ::write(fd, data, size);
```

Comment:
This one I would rather either keep the warning or implement a real fix (aka handle a failed write())

Reply:
Done